composer.json

```diff
@@ -27,7 +23,7 @@
         "typo3-ter/dyncss": ">=0.7.0 <=0.7.99",
         "typo3-ter/dyncss-less": ">=0.7.0 <=0.7.99",
         "typo3-themes/themes": ">=7.0.0 <=7.2.99",
-        "t3kit/t3kit-extension-tools": "*",
+        "typo3-ter/t3kit-extension-tools": "^1.0",
```

Comment:
yes, with require-dev section it will be more correct way.
```
"t3kit/t3kit-extension-tools": "*"
or
"t3kit/t3kit-extension-tools": "dev-master"
or it doesn't matter?
```